addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderThingMonitorHandler.java

```diff
@@ -893,14 +1194,28 @@ public String getLogIdentifier() {
 
         try {
             if (config != null) {
-
                 result = String.format("[MONITOR-%s]", config.getZoneMinderId().toString());
             }
 
         } catch (Exception ex) {
```

Comment:
Please catch specific exceptions instead of catching all exceptions. That way it is more clear what exceptions need to be handled and how this is done.